src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -573,6 +577,62 @@ namespace AirflowNetwork {
         }
     };
 
+    struct SpecifiedMassFlow : public AirflowElement // Specified mass flow element
+    {
+        // Members
+        Real64 mass_flow; // Mass Flow [kg/s]
+
+        // Default Constructor
+        SpecifiedMassFlow() : mass_flow(0.0)
+        {
+        }
+
+        int calculate([[maybe_unused]] EnergyPlusData &state,
+                      [[maybe_unused]] bool const LFLAG,           // Initialization flag.If = 1, use laminar relationship
+                      [[maybe_unused]] Real64 const PDROP,         // Total pressure drop across a component (P1 - P2) [Pa]
+                      [[maybe_unused]] int const i,                // Linkage number
+                      const Real64 multiplier,                     // Element multiplier
+                      const Real64 control,                        // Element control signal
+                      [[maybe_unused]] const AirProperties &propN, // Node 1 properties
+                      [[maybe_unused]] const AirProperties &propM, // Node 2 properties
+                      std::array<Real64, 2> &F,                    // Airflow through the component [kg/s]
+                      std::array<Real64, 2> &DF                    // Partial derivative:  DF/DP
+        );
+
+        virtual ComponentType type()
+        {
+            return ComponentType::SMF;
+        }
+    };
+
+    struct SpecifiedVolumeFlow : public AirflowElement // Specified mass flow element
+    {
+        // Members
+        Real64 volume_flow; // Volume Flow [m3/s]
+
+        // Default Constructor
+        SpecifiedVolumeFlow() : volume_flow(0.0)
+        {
+        }
+
+        int calculate([[maybe_unused]] EnergyPlusData &state,
+                      [[maybe_unused]] bool const LFLAG,   // Initialization flag.If = 1, use laminar relationship
+                      [[maybe_unused]] Real64 const PDROP, // Total pressure drop across a component (P1 - P2) [Pa]
+                      [[maybe_unused]] int const i,        // Linkage number
+                      const Real64 multiplier,             // Element multiplier
+                      const Real64 control,                // Element control signal
+                      const AirProperties &propN,          // Node 1 properties
+                      const AirProperties &propM,          // Node 2 properties
+                      std::array<Real64, 2> &F,            // Airflow through the component [kg/s]
+                      std::array<Real64, 2> &DF            // Partial derivative:  DF/DP
+        );
+
+        virtual ComponentType type()
+        {
+            return ComponentType::SVF;
+        }
+    };
+
```

Comment:
We ended up adding two separate objects to avoid explicit logical checks as to whether the object is supposed to be fixed mass flow or fixed volume flow.